### ISSUE TITLE
NPE in AboutActivity

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#12197]
### Describe your problem!

From Google Play console, happening in 2021.11.21 (but also in older versions):

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3520)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3692)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:85)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2142)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:236)
  at android.app.ActivityThread.main (ActivityThread.java:7864)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:620)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1011)
Caused by: java.lang.NullPointerException: 
  at cgeo.geocaching.AboutActivity$SystemViewCreator.setContent (AboutActivity.java:260)
  at cgeo.geocaching.activity.TabbedViewPagerFragment.notifyDataSetChanged (TabbedViewPagerFragment.java:61)
  at cgeo.geocaching.activity.TabbedViewPagerActivity.setOrderedPages (TabbedViewPagerActivity.java:110)
  at cgeo.geocaching.activity.TabbedViewPagerActivity.createViewPager (TabbedViewPagerActivity.java:49)
  at cgeo.geocaching.AboutActivity.onCreate (AboutActivity.java:102)
  at android.app.Activity.performCreate (Activity.java:8093)
  at android.app.Activity.performCreate (Activity.java:8067)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1309)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3493)
```

Occurrences: 305 by 299 users
Android versions 9-12
Various devices 

### How to reproduce?

Unclear

### Reproducible

Unclear

### c:geo Version

2021.11.21

**Comment by Lineflyer** [comment]
A similar stack trace but listed separate on Google Play:

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3520)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3692)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:85)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2142)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:236)
  at android.app.ActivityThread.main (ActivityThread.java:7864)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:620)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1011)
Caused by: java.lang.NullPointerException: 
  at cgeo.geocaching.AboutActivity$SystemViewCreator.setContent (AboutActivity.java:260)
  at cgeo.geocaching.activity.TabbedViewPagerFragment.notifyDataSetChanged (TabbedViewPagerFragment.java:61)
  at cgeo.geocaching.activity.TabbedViewPagerActivity.setOrderedPages (TabbedViewPagerActivity.java:110)
  at cgeo.geocaching.activity.TabbedViewPagerActivity.createViewPager (TabbedViewPagerActivity.java:49)
  at cgeo.geocaching.AboutActivity.onCreate (AboutActivity.java:102)
  at android.app.Activity.performCreate (Activity.java:8093)
  at android.app.Activity.performCreate (Activity.java:8067)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1309)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3493)
```

**Comment by eddiemuc** [comment]
Fixed with PR #12198
